cdap-client-tests/src/test/java/co/cask/cdap/client/QueryClientTestRun.java

```diff
@@ -69,7 +71,9 @@ public void testAll() throws Exception {
     Thread.sleep(3000);
 
     executeBasicQuery();
-    exploreClient.disableExploreDataset("cdap.default." + FakeApp.DS_NAME).get();
+    Id.DatasetInstance datasetInstance = Id.DatasetInstance.from(getClientConfig().getNamespace(),
```

Comment:
Any way to set a non-default namespace in the ``ClientConfig`` before this test, then construct the equivalent of ``cdap.default.`` String literal below using a combination of ``ClientConfig`` and ``CConfiguration``?